ThunderTable/TableRow+Actions.swift

```diff
@@ -0,0 +1,199 @@
+//
+//  TableRow+Actions.swift
+//  ThunderTable
+//
+//  Created by Simon Mitchell on 15/02/2018.
+//  Copyright © 2018 3SidedCube. All rights reserved.
+//
+
+import UIKit
+
+/// The action style for a contextual action
+///
+/// - `default`: Equivalent to normal on iOS >= 11
+/// - destructive: A destructive action, i.e. removes the row
+/// - normal: Any other action
+public enum RowActionableStyle {
+	case `default`
+	case destructive
+	case normal
+	
+	@available(iOS 11.0, *)
+	var _UIContextualActionStyle: UIContextualAction.Style {
+		switch self {
+		case .destructive:
+			return .destructive
+		default:
+			return .normal
+		}
+	}
+	
+	var _UITableViewRowActionStyle: UITableViewRowActionStyle {
+		switch self {
+		case .destructive:
+			return .destructive
+		case .default:
+			return .default
+		case .normal:
+			return .normal
+		}
+	}
+}
+
+/// A closure called when a `RowActionable` is triggered by the user
+/// call the completionHandler (iOS 11 and above) to reset the context to its normal state (e.g. when swiping, resets to unswiped state)
```

Comment:
If this is a separate sentence to the one above, could it start with a capital please? Got a bit confused as I thought it was one sentence to begin with :)